README.md

```diff
@@ -87,6 +92,92 @@ TODO:
 - ??
 
 
+### Packet / Message
+
+<!--
+src: https://github.com/ssbc/tinySSB/blob/fee99079ac2820711e1e853cba0e7aacc8765fea/android/tinySSB/app/src/main/java/nz/scuttlebutt/tremolavossbol/tssb/Replica.kt#L378-L403
+-->
+
+A "packet" 120 Bytes of data which consists of the following data concatenated in order:
+
+name          | bytes
+:-------------|:------
+`dmx`         | 7
+`packet_type` | 1
+`content`     | 48 (or more if chunked)
+`signature`   | 64
+
+For `content` up to 48 bytes in size, a single packet suffices, for more than that, see "side-chains" later.
+
+
+
+
+#### DMX
+
+The DMX is a header designed to encode all the information required to build up
+a feed (signed linked list), while avoiding having to explicitly include the
+full `feed_id`, `sequence`, and `prev_message_id` in the message. Instead of linking back with these 3 data points, we link back to the *hash* of those. 
+
+e.g. This means that if you have message 23 for a feed, you will be able to calculate the expected DMX for message 24. When you receive a new packet you can check to see if it matches the DMX for the "next message" in any feed you're tracking.
+
+The DMX is defined as the first 7 bytes of the SHA256 hash of the following
+data concatenated in order:
+
+name              | details
+:-----------------|:-----------
+`dmx_prefix`      | (default: `tinyssb-v0`)
+`feed_id`         | encoded as ???
+`sequence`        | the sequence of this packet in the feeds linked-list
+`prev_message_id` | the id of the previous message in this feeds linked-list
```

Comment:
I don't know how this this `prev_message_id` is calculated yet. I assume it's the hash of ... something, but haven't read it in the code yet.